### ISSUE TITLE
contr+shift+i not give to codegpt the actual content of the comment on c#

### ISSUE THREAD

**Issue by MarcoLavoro** [JudiniLabs/code-gpt-docs#233]
How you can see from the image if I press contr+shift+i at the end of the comment it send to codegpt: 
Explain, write and ALWAYS print the result of the code. Use markdown format and use the following code: \n\n

and NOT the content of the comment.
Is a bug right? I not found a way to fix it
I am on c#

![1Capture](https://github.com/davila7/code-gpt-docs/assets/42266887/f2bea7a4-b880-43b4-a2ab-1ad8a220a212)
![server message](https://github.com/davila7/code-gpt-docs/assets/42266887/806612b8-aeb7-4b16-8e25-4e5270587d6d)

**Comment by PilarHidalgo** [comment]
@MarcoLavoro  Hi!, yes you right, to connect CodeGPT or any provider no need to use CMD+SHFT+P command to set the connection anymore.

**Comment by PilarHidalgo** [comment]
@MarcoLavoro Yes you right, to use the features no need to execute the command anymore, please check the docs here: https://docs.codegpt.co/docs/tutorial-features/code_explanation

**Comment by MarcoLavoro** [comment]
> @MarcoLavoro Yes you right, to use the features no need to execute the command anymore, please check the docs here: https://docs.codegpt.co/docs/tutorial-features/code_explanation

> @MarcoLavoro Yes you right, to use the features no need to execute the command anymore, please check the docs here: https://docs.codegpt.co/docs/tutorial-features/code_explanation

What I meant is that I expect that by pressing contr+shift+i at the end of the comment line, codegpt takes the comment line value (In this case: "create a unity 3d function to instantiate a cube") and send it to the LLM

But instead, the codegpt just sends the generic: 
Explain, write and ALWAYS print the result of the code. Use markdown format and use the following code: \n\n

**Comment by PilarHidalgo** [comment]
@MarcoLavoro  thanks!
